From e58226a399be7cf2cc6d91b66facb65d7dfb5018 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Tue, 20 Aug 2019 15:40:18 +0800
Subject: [PATCH] Supply firmware at build time

We'd like to supply firmware when fetching the sources. Disable runtime
fetching and point to fetched firmware instead.

Upstream-Status: Submitted

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>

---
 inference-engine/cmake/dependencies.cmake | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/inference-engine/cmake/dependencies.cmake b/inference-engine/cmake/dependencies.cmake
index 64435ca..217fbc9 100644
--- a/inference-engine/cmake/dependencies.cmake
+++ b/inference-engine/cmake/dependencies.cmake
@@ -37,24 +37,6 @@ else()
     set(MODELS_BRANCH "master")
 endif()
 
-if (ENABLE_MYRIAD)
-    RESOLVE_DEPENDENCY(VPU_FIRMWARE_MA2450
-            ARCHIVE_UNIFIED firmware_ma2450_491.zip
-            TARGET_PATH "${TEMP}/vpu/firmware/ma2450"
-            ENVIRONMENT "VPU_FIRMWARE_MA2450"
-            FOLDER)
-    debug_message(STATUS "ma2450=" ${VPU_FIRMWARE_MA2450})
-endif ()
-
-if (ENABLE_MYRIAD)
-    RESOLVE_DEPENDENCY(VPU_FIRMWARE_MA2480
-            ARCHIVE_UNIFIED firmware_ma2480_mdk_R7_9.zip
-            TARGET_PATH "${TEMP}/vpu/firmware/ma2480"
-            ENVIRONMENT "VPU_FIRMWARE_MA2480"
-            FOLDER)
-    debug_message(STATUS "ma2480=" ${VPU_FIRMWARE_MA2480})
-endif ()
-
 ## enable cblas_gemm from OpenBLAS package
 if (GEMM STREQUAL "OPENBLAS")
 if(NOT BLAS_LIBRARIES OR NOT BLAS_INCLUDE_DIRS)
